src/docs/product/performance/resources.md

```diff
@@ -138,6 +138,20 @@ If you want to investigate a specific page, click on it to open a sidebar showin
 
 <div style="position: relative; padding-bottom: calc(50.829268292682926% + 41px); height: 0;"><iframe src="https://demo.arcade.software/D96Pu2LMLHddyVCn3BOd?embed" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;color-scheme: light;"></iframe></div>
 
+### Image Samples
+
+When viewing an image resource, the resource summary page will also show the 5 largest image samples. Sentry determines the span is an image if it has one of the following extensions: 'jpg', 'jpeg', 'png', 'gif', 'svg', 'webp', 'apng', 'avif'.
+
+If size information is not available, the samples will not be ordered in any particular order. If the images are publicly accessible and the 'Enable Images' project settings is enabled, then the actual images will be rendered in the UI.
+
+<Note>
+
+At the moment, relative resource span descriptions (ones that begin with '/') are not supported for the image preview. We are working on adding this soon!
```

Comment:
We should open a GH issue and link it here for the feature request - we tend to use this pattern for stuff like this.

Even though we know we are going to fix it, it might also be another avenue to encourage general feedback.